isisd/isis_tlvs.c

```diff
@@ -2613,24 +2647,24 @@ static void format_item_prefix_sid(uint16_t mtid, struct isis_item *i,
 			((sid->flags & ISIS_PREFIX_SID_LOCAL) ? "yes" : ""));
 		/* end deprecated keys (no booleans) */
 
-		struct json_object *flags_json;
+		struct json_object *srv6_end_sid_json;
```

Comment:
Here we are showing Prefix SIDs of SR-MPLS (not SRv6).

Reply:
Yes, it's the prefix SID, but this time it's the correct name. It is displaying the flags
I have kept `flags_json`